resources/views/layout.blade.php

```diff
@@ -1,23 +1,8 @@
 @php
-use Illuminate\Support\Facades\Vite;
-use Illuminate\Foundation\Vite as ViteFoundation;
+$nonExistentFileName = public_path('/vendor/horizon/nonExistentFile');
+$previousHotFile = Vite::hotFile();
```

Comment:
We save the previously detected hotfile of the user so we can reset the hotfile at the end of the blade template and "restore" the previous state.